### PR TITLE
First draft of newsletter 28

### PR DESCRIPTION
**Pull request by SPrinss**
Meatless meatballs not included. Could be added?

### CODE REVIEW

**Comment by SPrinss** [comment]
Still miissing:
- the constructive journalism bit (about meatless meat?)

File: newsletters/2015-12-25-28.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+published: true
+subject: Meatless meatballs, electricity from cheese,
+teaser: 🌲🔋🎊
+background: 2015-12-12-26-party.gif  
+---
+# Hello!
+
+This week an issue is all about energy -- and a big rocket!
```
Comment (thomasdebeus) [inline newsletters/2015-12-25-28.md]: -is

Comment (SPrinss) [inline newsletters/2015-12-25-28.md]: Not necessarily .. Deze week een uitgave over een bakker die brood bakt.

File: newsletters/2015-12-25-28.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This cheese manufacturer decided to partner up with the local power plant to use excess skimmed whey -- an unneeded by-product of making the cheese -- as power for 1500 homes. How cool 😎is that?
+
+The energy comes from the bacteria that find the whey delicious and produce methane, a biogas, in return. Thanks little guys!
+
+[eddie.net](http://www.edie.net/news/5/French-power-station-generates-electricity-for-1-500-from-cheese/)
+
+![rubbishbin](2015-12-25-28-bin.jpg)
+
+There is a lot of waste in the oceans -- continent sized collections of small plastics and lot's of oil. These surfer guys bumped into so much waste while surfing that they decided to create 'rubbish bins' to catch the it!
```
Comment (thomasdebeus) [inline newsletters/2015-12-25-28.md]: Maybe remove that their is so much waste in the ocean people no this by now...

Yo can keep the part that they;re surfers and they bumped in to it though

File: newsletters/2015-12-25-28.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+The bins pump in the surrounding water, trapping waste inside the bin. The plastics caught in the bins can be re-used to create more bins. Awesome! You can also put a bin on your solar powered yacht, cleaning the planet while relaxing; doesn't that sound great?
```
Comment (thomasdebeus) [inline newsletters/2015-12-25-28.md:34]: The floating bins perhaps?

**Comment by thomasdebeus** [comment]
Oh and titles on each piece are missing

**Comment by bvhme** [comment]
I like this issue, very good:smile:!

**Comment by bvhme** [comment]
Oh, and the background image for the header needs to be chosen!

**Comment by thomasdebeus** [comment]
Merge?

**Comment by bvhme** [comment]
:+1:

On Thu, 24 Dec 2015 at 10:43 thomas notifications@github.com wrote:

> Merge?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/newatoms/hwhbstptw/pull/19#issuecomment-167088452.
> 
> ## 

Boris van Hoytema

—
- E-mail boris@vanhoytema.com
- Website bvh.me
- Twitter twitter.com/bvhme
- Facebook facebook.com/vanhoytema
- LinkedIn linkedin.com/in/borisvanhoytema

**Pull request merged**